.papr.yml

```diff
@@ -55,6 +55,8 @@ env:
 # present, tests will run after a successful build. Full
 # UTF-8 values are supported.
 tests:
+    - yum install -y --skip-broken pylint
+    - yum downgrade -y pylint
```

Comment:
This is the sort of thing where I like to add big screaming comments explaining what's going on, why, and when it should be time to revisit / review / remove this. Likewise for the commented-out pylint above. Aside from that, LGTM but we'll have to see if it works.